gui/src/components/QuickAddressSelect.tsx

```diff
@@ -29,6 +29,7 @@ export function QuickAddressSelect() {
   return (
     <Select
       key={currentWallet.name}
+      value={getCurrentPath(currentWallet, addresses)}
```

Comment:
I don't have enough context to remember, can you explain why this is needed?
Seeing both `value` and `defaultValue` as props seems weird

Reply:
setting `value` is needed to update the displayed value when we change the address from the command bar, otherwise it wouldn't update

Reply:
no, what you need is to force a re-render of the react object
setting `value` explicitly will conflict with the ability to use the input itself

possibly what you want is to set a `key={...}` to make sure it re-renders that component when some outside value change, but still have it react to updates in the select itself

Reply:
from what I tested, it doesn't seem to conflict with anything, I can still change the value from clicking the ui or by using the command bar

Reply:
ok, changing the `key` value from `currentWallet.name` to getting the current path seems to also work